Algorithm.Python/KerasNeuralNetworkAlgorithm.py

```diff
@@ -0,0 +1,97 @@
+# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import clr
+clr.AddReference("System")
+clr.AddReference("QuantConnect.Algorithm")
+clr.AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Algorithm import *
+
+import numpy as np
+from keras.models import Sequential
+from keras.layers import Dense, Activation
+from keras.optimizers import SGD
+
+class KerasNeuralNetworkAlgorithm(QCAlgorithm):
+
+    def Initialize(self):
+        self.SetStartDate(2013, 10, 7)  # Set Start Date
+        self.SetEndDate(2013, 10, 8) # Set End Date
+        
+        self.SetCash(100000)  # Set Strategy Cash
+        spy = self.AddEquity("SPY", Resolution.Minute)
+        self.symbols = [spy.Symbol]
```

Comment:
Since we are using only want symbol, SPY, why is this list defined?
I would suggest:
```python
self.symbol = self.AddEquity("SPY", Resolution.Minute).Symbol
```
and replace every `"SPY"` occurence for `self.symbol`.
Also, it would replace all loops and dictionaries below.

On the other hand, another security could be added:
```python
qqq = self.AddEquity("QQQ", Resolution.Minute)
self.symbols = [spy.Symbol qqq.Symbol]
```

Reply:
provided the rest of the algorithm can handle a list, then I think it's nice to use list notation as it makes it easier for users to extend for multiple symbols.

Reply:
Thanks for all your comments. My idea is that using a list is much easier for users to further this strategy when they need to include more trading symbols.